src/plugins/visualize/public/application/utils/breadcrumbs.ts

```diff
@@ -21,16 +21,8 @@ import { i18n } from '@kbn/i18n';
 
 import { VisualizeConstants } from '../visualize_constants';
 
-const appPrefixes: Record<string, any> = {
-  dashboards: {
-    text: i18n.translate('visualize.dashboard.prefix.breadcrumb', {
-      defaultMessage: 'Dashboard',
-    }),
-  },
-};
-
 const defaultEditText = i18n.translate('visualize.editor.defaultEditBreadcrumbText', {
```

Comment:
This should be deleted from the translations files in order to be translated again

Reply:
@dej611 and I had a discussion about this a couple months ago, and I was under the impression that only unused translations should be removed, and it was easier for the translators if the old translation is kept in place.